src/less/tooltip/ds6/tooltip.less

```diff
@@ -35,8 +35,12 @@
 button.tourtip__close {
     color: @color-white;
 
-    &:focus {
+    &:focus,
+    &:hover {
```

Comment:
Is this the right color for hover? It does not change at all so it doesn't give any indication unlike infotip

Reply:
Hmm, good point. Let me check..

Reply:
Looks like it is meant to be always white

![Screen Shot 2020-02-04 at 9 49 40 AM](https://user-images.githubusercontent.com/38065/73771711-b42d6700-4733-11ea-97f2-2076e817efa7.png)